src/app/containers/Caption/index.jsx

```diff
@@ -7,8 +7,9 @@ import { ServiceContext } from '#contexts/ServiceContext';
 import Blocks from '../Blocks';
 import Fragment from '../Fragment';
 import InlineLink from '../InlineLink';
+import inline from '../InlineContainer';
```

Comment:
```suggestion
import Inline from '../InlineContainer';
```

Reply:
Has been implemented as `inline` throughout the code base maintained it as so for consistency https://github.com/bbc/simorgh/blob/9a5daab7ad8777aa2f0b9350440968f82abfce77/src/app/containers/Paragraph/index.jsx#L7

Reply:
🤔that must have been left in there by mistake, please update that too.